pulsar-client-cpp/include/pulsar/InitialPosition.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#ifndef PULSAR_CPP_INITIAL_POSITION_H
+#define PULSAR_CPP_INITIAL_POSITION_H
+
+#pragma GCC visibility push(default)
+namespace pulsar {
+enum InitialPosition
+{
+    Latest,
+    Earliest
```

Comment:
Unlike in Java, the enum values are not created in the enum scope, so it might be better to repeat that, eg: `InitialPositionLatest`. 

Also, "earliest" should come before "latest"

Reply:
@merlimat in PulsarApi.proto, the "latest" should come before "latest". 
 [PulsarApi.proto](https://github.com/apache/pulsar/blob/master/pulsar-common/src/main/proto/PulsarApi.proto#L232)